server/build.go

```diff
@@ -607,11 +603,6 @@ esbuild:
 						DevMode:      task.DevMode,
 					}
 					subTask.treeShaking = newStringSet()
-					_, err = subTask.build(tracing)
```

Comment:
I think we should keep this, build sub-modules from scratch is slow even you keep the lock

Reply:
But it will block the response, I think we can pre-build sub-modules while the main module build done.

Reply:
are you sure the final import time is optimized if build sub-modules from request? if yes, try to increase the timeout

Reply:
Seems the timeout problem may be due to locks, the tfjs pkg will show the problem,  debugging